nims-notes-things/src/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components";
+
+const Button = (props) => {
+  return <ButtonStyle {...props}>{props.children}</ButtonStyle>;
```

Comment:
This works but because you are passing a background prop to the styled components, I would specify it in the props. Just makes it more clear.

```
<ButtonStyle 
   background={props.background} 
   {...props}
>
      {props.children}
</ButtonStyle>
```